PluginFramework/src/Update/BaseUpdate.cs

```diff
@@ -0,0 +1,49 @@
+﻿using PluginFramework.src.DataContainer;
+using PluginFramework.src.Interfaces.PluginTypes;
+using System;
+using System.Collections.Generic;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5528129)